src/oidcop/token/handler.py

```diff
@@ -147,20 +147,42 @@ def factory(
     TTYPE = {"code": "A", "token": "T", "refresh": "R"}
 
     key_defs = []
+    read_only = False
+    if kwargs.get('jwks_def'):
+        defs = kwargs['jwks_def']
+        jwks_file = defs.get('private_path', jwks_file)
+        read_only = defs.get('read_only', read_only)
+        key_defs = defs.get('key_defs', [])
+
+    for _keyd in key_defs:
+        if _keyd['kid'] == 'code':
+            code = _keyd
+        elif _keyd['kid'] == 'refresh':
+            refresh = _keyd
+        elif _keyd['kid'] == 'token':
+            token = _keyd
+
     if code is not None:
```

Comment:
@rohe 
should it be `if not code` instead?

if these definitions already have been configured in the general conf, why should we have to overwrite them?
they would be created if those arguments are None and also configuration's key_defs are None.

Do you agree?
If it's the way Give me a sign of this, I'll push a PR for this

Reply:
Ok, you've already achieved this here
https://github.com/IdentityPython/oidc-op/commit/062dc1dff66d215b698bff0bb8697f8d653e1d57

well done.